File: .ccls-cache/@home@runner@Beecrowd-BeginerParte-2/1049.c
#include <stdio.h>
#include <string.h>

int main(){

char x[100] , y[100] , z[100];

  scanf("%s\n%s\n%s" , x , y , z);

if(strcmp(x, "vertebrado") == 0){

if(strcmp(y, "ave") == 0){
  if(strcmp(z, "carnivoro") == 0)
    printf("aguia\n");
  if(strcmp(z, "onivoro") == 0)
    printf("pomba\n");
}
  if(strcmp(y, "mamifero") == 0){
    if(strcmp(z, "onivoro") == 0)
      printf("homem\n");
    if(strcmp(z, "herbivoro") == 0)
      printf("vaca\n");
    
  }
    
}

  if(strcmp(x , "invertebrado") == 0){
    if(strcmp(y , "inseto") == 0){
      if(strcmp(z , "hematofago") == 0)
        printf("pulga\n");
      if(strcmp(z , "herbivaro") == 0) 
        printf("lagarta\n");
    }

    if(strcmp(y , "anelideo") == 0){
      if(strcmp(z , "hematofago") == 0)
        printf("sanguessuga\n");
      if(strcmp(z , "onivoro") == 0)
        printf("minhoca\n");
    }
  }




  return 0;
}

File: 1049.py
# x = input()
# y = input()
# z = input()

# if x == "vertebrado" and y == "ave" and z == "carnivoro":
#   print("aguia\n")

# elif x == "vertebrado" and y == "ave" and z == "onivoro":
#   print("pomba\n")

# elif x == "vertebrado" and y == "mamifero" and z == "onivoro":
#   print("homem\n")

# elif x == "vertebrado" and y == "mamifero" and z == "herbivoro":
#   print("vaca\n")

# elif x == "invertebrado" and y == "inseto" and z == "hematofago":
#   print("pulga\n")

# elif x == "invertebrado" and y == "inseto" and z == "herbivoro":
#   print("lagarta\n")

# elif x == "invertebrado" and y == "anelideo" and z == "hematofago":
#   print("sanguessuga\n")

# elif x == "invertebrado" and y == "anelideo" and z == "onivoro":
#   print("minhoca\n")

x = input()
y = input()
z = input()

if(x == "vertebrado"):
    if(y == "ave"):
        if(z == "carnivoro"):
            print("aguia\n")
        elif(z == "onivoro"):
            print("pomba\n")
    elif(y == "mamifero"):
        if(z == "onivoro"):
            print("homem\n")
        elif(z == "herbivoro"):
            print("vaca\n")
elif(x == "invertebrado"):
    if(y == "inseto"):
        if(z == "hematofago"):
            print("pulga\n")
        elif(z == "herbivoro"):
            print("lagarta\n")
    elif(y == "anelideo"):
        if(z == "hematofago"):
            print("sanguessuga\n")
        elif(z == "onivoro"):
            print("minhoca\n")
        
            
        